sql-database/create-and-configure-database/create-and-configure-database.sh

```diff
@@ -1,47 +1,56 @@
 #!/bin/bash
 
+# set execution context (if necessary)
+az account set --subscription <replace with your subscription name or id>
+
+# Set the resource group name and location for your server
+export resourceGroupName=myResourceGroup-$RANDOM
```

Comment:
Avoid exporting script-local values into the shell, especially since they may overwrite user variables.

Reply:
@sptramer I do not understand what you are asking here

Reply:
Don't use the `export` keyword.

Reply:
this is how I was taught to create variables in bash

Reply:
removed export